scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1346,5 +1378,5 @@
        (resource-offers [this driver offers]
                         (receive-offers offers-chan driver offers))
        (status-update [this driver status]
-                      (future (handle-status-update conn driver fenzo status))))
+                      (handle-status-update conn driver fenzo status)))
```

Comment:
Wil: The reason the future is there is actually a classloader problem. For
reasons I don't know how to investigate, the thread the mesos client
launches on doesn't pick up the full class path but launching a new thread
does. Last time I say the problem it showed up with an exception
clojure/Lang/reflector not found.

@wyegelwel  Did this used to happen on the integration/simulation tests also? Or did it happen only in production?